hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -161,9 +164,13 @@ public void addDeserializedChildren(List<MerkleNode> children, int version) {
 	@Override
 	public void migrate() {
 		int deserializedVersionFromState = getDeserializedVersion();
-		if (deserializedVersionFromState < RELEASE_0220_VERSION) {
-			blobMigrator.migrateFromBinaryObjectStore(this, deserializedVersionFromState);
-			init(getPlatformForDeferredInit(), getAddressBookForDeferredInit(), getDualStateForDeferredInit());
+		if (deserializedVersionFromState < RELEASE_0240_VERSION) {
+			stakeFundingMigrator.accept(this);
+			if (deserializedVersionFromState < RELEASE_0220_VERSION) {
+				// Remove after confirming no regression tests use 0.21.x states
+				blobMigrator.migrateFromBinaryObjectStore(this, deserializedVersionFromState);
+				init(getPlatformForDeferredInit(), getAddressBookForDeferredInit(), getDualStateForDeferredInit());
```

Comment:
I guess we can remove these and make sure regression is updated to latest state ?

Reply:
If there are no objections, I will remove! Mirror node team has asked for an enhancement to stream synthetic `CryptoCreate` records for these two accounts, so I need to push again anyways. 😁

Reply:
Sounds good .. 😄